Source/console.h

```diff
@@ -11,7 +11,7 @@
 #pragma once
 
 #include "zerochecker.h"
-#include <CppConsoleTable.hpp>
+#include "CppConsoleTable.hpp"
```

Comment:
See my earlier note. This header include is intentionally bracketed, as it's an external library file in the /Vendor/ submodule.